cfpb/cfgov-refresh
# Fix event livestream

This fixes an issue we were seeing where the event stream wasn't going live until an hour after the start date of the event.

We are now using the event's go live date instead of the event's start time, treating it as an ET time, and comparing it to the current time in ET to determine if it's past, present, and future (along with the end time).

Eventually we will need to clean up the app from converting things to ET but that relies on a fix here: https://github.com/torchbox/wagtail/issues/2406

To test:
Edit an event page to have a livestream, and set the start time to the current time in ET (or close to it). Confirm that it shows the livestream if it's past and does not if it's future.

Also, you can run the test:
`Run tox -- v1.tests.test_event_state`

@jimmynotjim @kave @kurtw @rosskarchner
